LambdaRuntimeDockerfiles/Images/net6/amd64/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Based on Docker image from: https://github.com/dotnet/dotnet-docker/
 
-ARG ASPNET_VERSION=6.0.11
-ARG ASPNET_SHA512=12a30719aacd5b3dd444d075c13966a4bb1dc149c36bcbc0e685730f08d1c75eb3929749b89a88569ddb48bd8104db84aaee2ee097ac3a5fe6fff60c9f09f741
+ARG ASPNET_VERSION=6.0.12
```

Comment:
Should we go to 6.0.13 instead?
* https://github.com/dotnet/core/blob/main/release-notes/6.0/6.0.13/6.0.13.md?WT.mc_id=dotnet-35129-website
* https://dotnetcli.blob.core.windows.net/dotnet/checksums/6.0.13-sha.txt

Reply:
Good catch. Update the PR to point to 6.0.13